mindjunkies/courses/views.py

```
@@ -277,3 +277,20 @@
         course.update_rating()
 
         return redirect(reverse("course_details", kwargs={"slug": course.slug}))
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)